src/Databases/Sqlite.php

```diff
@@ -18,8 +18,16 @@ public function dumpToFile(string $dumpFile): void
 
     public function getDumpCommand(string $dumpFile): string
     {
-        $includeTables = rtrim(' '.implode(' ', $this->includeTables));
-
+        $includeTables = rtrim(' ' . implode(' ', $this->includeTables));
```

Comment:
I don't understand what is happening here, seems a bit hacky. If this is because of possible empty values, use array_filter to remove them before imploding.

Reply:
that `implode` line of code was there before my PR and I just didn't change it so everything doesn't break. 

i've just updated the `includeTables` based on the excluded ones.

Reply:
ok, `rtrim` is there to remove the first whitespace that is concated to `implode` when the `includeTables` are empty. and that is necessary to pass a lot of tests.